### ISSUE TITLE
impossible to install on Ubuntu 20.04

### ISSUE THREAD

**Issue by s4dic** [damentz/liquorix-package#136]
Since today its impossible to install on 20.04  :
```
Reading package lists... Done

[INFO ] Liquorix PPA repository added successfully

Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package linux-image-liquorix-amd64
E: Unable to locate package linux-headers-liquorix-amd64
```
Why? Is there a valid reasons for this ?

**Comment by damentz** [comment]
Ubuntu Focal is missing packages required to build Linux 6.4 with perf support: https://github.com/damentz/liquorix-package/commit/5eae49c0737cee9d68369c6c1725a07befb0dea8

Also note, 20.04 is now over 3 years old.  The chance Liquorix will work well on a desktop using this distribution are very low since the user space is so out dated.  I recommend you upgrade to the next LTS (Ubuntu Jammy 22.04), if you desire long term support.

**Comment by s4dic** [comment]
Yes I understand, but why I can't install the last package of kernel with ppa ?

**Comment by damentz** [comment]
It failed to build and previous packages were removed automatically.  I actually just put in a request to delete that package along with the last Kinetic, so they won't be there shortly.

Like I mentioned, I **_highly_** recommend you upgrade; new kernel with ancient distribution is never a good combination.

**Comment by damentz** [comment]
Closing issue, Focal is simply not supported anymore and I don't recommend continued use with a modern Liquorix kernel.

**Comment by damentz** [comment]
FYI, I built and published a 6.3.13 kernel for Ubuntu Focal / 20.04 users through the PPA.  Remember though, this is just a workaround and the kernel has not been tested by me, it's is simply available as is.
